### ISSUE TITLE
Add yen self update

### ISSUE THREAD

**Issue by lbrealdev** [tusharsadhwani/yen#31]
Hi @tusharsadhwani,

I've been using yen for months, and really, always very happy with each release.

However, I see that over time, yen is becoming more popular, and this could mean more releases, improvements, fixes and etc.

Therefore, I come through this issue, requesting a significant improvement for the yen, which would be the possibility of the yen being self-updated.

Something like that:
```shell
yen self update
```

or

```shell
yen update
```

Would it be possible to make this feature available in future versions ?

I await your feedback.

Regards.

**Comment by tusharsadhwani** [comment]
Definitely, this is possible. Fairly simple to make too. I'll try to do it over the weekend.
